File: stockservice/src/main/java/com/api/stockservice/domain/event/Sortie/SortieItem.java
package com.api.stockservice.domain.event.Sortie;

import com.api.stockservice.domain.Entities.Product;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class SortieItem {
    private String ProductId;
    private int Quantity;
}

File: backend/services/MarketplaceService/src/MarketplaceService.Domain/Events/Sortie/SortieItem.cs
using EventsContracts.EventsContracts;

namespace MarketplaceService.Domain.Events.Sortie
{
    public class SortieItem : ISortieItem
    {
        public string ProductId { get; set; }
        public int Quantity { get; set; }
        public double Price { get; set; }
    }
}
